frontend/src/app/services/base.service.ts

```diff
@@ -18,6 +18,10 @@ export class BaseService {
     public post(url: string, data: Object) {
         return this.http.post(`${this.ROOT_URL}/${url}`, data);
     }
+
+    public auth(url: string, data: Object) {
```

Comment:
Do we have a PR for implementation for `auth`? Couldn't find one.

Reply:
Not necessary. I merged the existing `login` and `signup` methods into `auth` since they are the same. Just different route paths. The routing in the backend has already been set up as part of the old template code